ACPs/23-p-chain-native-transfers.md

```diff
@@ -0,0 +1,58 @@
+```text
+ACP: <PR Number>
+Title: P-chain native transfers
+Author(s): Dhruba Basu <https://github.com/dhrubabasu>
+Discussions-To: <GitHub Discussion URL>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+Support native transfers on P-chain.
```

Comment:
Can you add some context here in terms of who will be using the new features, or what new use cases are unlocked.